src/firefly/java/edu/caltech/ipac/firefly/data/TableServerRequest.java

```diff
@@ -30,6 +30,7 @@
     public static final String INCL_COLUMNS = "inclCols";
     public static final String FIXED_LENGTH = "fixedLength";
     public static final String META_INFO = "META_INFO";
+    public static final String META_OPTIONS = "META_OPTIONS";
```

Comment:
Just curious, why you decided to introduce META_OPTIONS rather than using META_INFO? META_INFO is something that is not supposed to be returned to the client, while META_OPTIONS are not returned? So the new highlighted Idx will be returned in META_INFO?

Reply:
META_INFO is returned to the client as part of the table's request and is used whenever you need to modify the table.  I introduce META_OPTIONS to pass along information that is needed once and should not be used in subsequent requests.  So yes, META_INFO is returned to the client while META_OPTIONS is not.